docs/configuration.md

```diff
@@ -0,0 +1,87 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution is using a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
+files which may exist in the project's repository and performs scans based solely
+on Precaution specific configuration.
+
+Precaution provides you with the option to use optional `.precaution.yaml` file.
+This file **must** be in the root directory of your project.
+
+All paths in the options should be **absolute**.
+
+This file can be configured with the following options:
+
+| Option | Syntax |
+|---|---|
+| [Exclude](#exclude) | [Globs](#globs) |
+
+## Exclude
+
+This option can be used to tell Precaution to exclude files or folders from processing.
+
+You should use the [Globs](#globs) syntax when defining the exclude option.
+
+If you want to ignore `src/cmd/main.go` file you will add this to your configuration file:
+```
+exclude:
+- "src/cmd/main.go"
+```
+
+Another use-case is when you are using Golang and your dependencies are in
+the `vendor/` folder, then you can tell Precaution to skip all files
+within the `vendor/` folder like this:
+```
+exclude:
+- "vendor/**"
+```
+
+It's possible that you wouldn't want Precaution to scan your Golang test files and if the suffix in their names is `_test.go`
+then you can easily exclude them:
+```
+exclude:
+- "*_test.go"
+```
+
+The exclude option in the `.precaution.yaml` file supports multiple
+exclude rules:
+```
+exclude:
+- "*_test.go"
+- "vendor/**"
+- "src/cmd/main.go"
+```
+
+## Globs
+
+| Pattern | Meaning
+|---|---|
+| `*` | Matches everything except slashes. |
```

Comment:
Forward slashes and backslashes?

Below you mention explicitly '/', so it's unclear here

Reply:
I change it and I rearranged the place of the `*` symbol to be underneath the `?` symbol.